database/buildSchema/16_template_element.sql

```diff
@@ -7,6 +7,7 @@ CREATE TYPE public.template_element_category AS ENUM (
 CREATE TYPE public.is_reviewable_status AS ENUM (
     'ALWAYS',
     'NEVER',
+    'ONLY_IF_APPLICANT_ANSWER',
```

Comment:
Why do we need this extra value? Currently the default value `NULL` does exactly this. Why change it?

We'd have to change all existing templates if we do, so would need to be a pretty strong case for why it's necessary.

Reply:
Yes, so the case is that otherwise the query for getting the review_response associated gets a lot more complex to deduce when we need to check the `is_reviewable_status` when it can be NULL but the application_response not NULL or if it's != 'ALWAYS' and application_response = NULL. Please have a look on the function `reviewable_questions` for more details.

Reply:
https://github.com/openmsupply/conforma-server/pull/867/commits/e4f7ed6071f837b3de5857cc61c8f480a2c35e33#r980706668

Reply:
Okay, in that case, can you make `ONLY_IF_APPLICANT_ANSWER` the `DEFAULT` and make the field `NON_NULLABLE`? So it'll behave the same as currently, but just with a value instead of null. (Might need to run migration code to change all the existing `null`s first.)

Will that require any front-end updates? Probably, right -- but Typescript should catch them once you re-run the codegen.

Reply:
![Screen Shot 2022-09-30 at 4 36 03 PM](https://user-images.githubusercontent.com/5456533/193184725-2ccb1978-7988-445c-bba2-471587efea92.png)

I don't think you did migration code for this, cos I'm still seeing `NULL` values after running migration script